jsonschema-core.xml

```diff
@@ -1510,22 +1510,15 @@
                         <t>
                             Together with "$dynamicAnchor", "$dynamicRef" implements a cooperative
                             extension mechanism that is primarily useful with recursive schemas
-                            (schemas that reference themselves).  The extension point is defined with
-                            "$dynamicAnchor" and only exhibits runtime dynamic behavior when referenced
-                            with "$dynamicRef".
+                            (schemas that reference themselves).  The extension point is defined
+                            with "$dynamicAnchor", and only exhibits runtime dynamic behavior when
+                            referenced with "$dynamicRef".
```

Comment:
In order to remove the "`$dynamicAnchor` produces a `$ref`-able anchor" behavior we, need to adjust this sentence.  We'll likely need to make changes in the `$dynamicAnchor` section as well.